README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <a href="https://www.gatsbyjs.com/?utm_source=starter&utm_medium=readme&utm_campaign=minimal-starter">
```

Comment:
This totally escaped me. I guess this is what you get with starter kits.

Reply:
At least it gave me something to actually change to get a new release!